LLama/Extensions/SpanNormalizationExtensions.cs

```diff
@@ -81,6 +81,19 @@ public static Span<float> EuclideanNormalization(this Span<float> vector)
         return vector;
     }
 
+    /// <summary>
+    /// <b>In-place</b> divide every element by the euclidean length of the vector
```

Comment:
This one isn't in-place. I assume this is required because one of the new APIs only gives you a `ReadOnlySpan`, so it can't be implemented in place?

Reply:
Yup (or, rather, a `ReadOnlyMemory<T>`, but yea). Fixing comment.